src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a list command. Upon execution, produces some feedback to the user.
+ */
+public class ListCommand extends Command {
+    /**
+     * Create instance of ListCommand.
+     */
+    public ListCommand() {
+
+    }
```

Comment:
if the constructor is empty, you can just not include it. its redundant.

Reply:
Yap, noticed. Thanks for the suggestion.